docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,46 +1,45 @@
 [[run-digger-installer]]
 === Run Digger Installer
 
-
 . Clone the https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^]
 
 . Make a copy of the Ansible https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[inventory-sample^], and update it with the information about your infrastructure.
 
 +
 NOTE: For a full list of variables that can be configured via the inventory file refer to <<installer-variables>>
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal:
+. Run the playbook and follow the instructions printed in the terminal:
 +
 NOTE: Refer to https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^] to see the roles executed during installation and their associated tags.
 
-** Command to install Digger without a macOS server
-+
-----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx
-----
-
 ** Command to install Digger with a macOS server
 +
 ----
 ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 ----
-
-** Roles can be run individually or in batch with flag --tags=<role_tag_1>,...,<role_tag_x>
-+
-----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --tags=provision-osx,nagios
-----
-** To skip default roles, similarly to --skip-tags=provision-osx e.g. --skip-tags=<role_tag_1>,...,<role_tag_x>
 +
-----
-ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=provision-osx,nagios,java
-----
+//** Command to install Digger without a macOS server
```

Comment:
should this line be commented out?

Reply:
yes, for now. noted in https://issues.jboss.org/browse/RHMAP-18949

resolved

Reply:
this actually works, readd it

todo

Reply:
resolved